styles.css

```diff
@@ -69,3 +110,135 @@
 .contact-info-button {
   margin-right: 30px;
 }
+
+/* Recent works */
+.recent-works {
+  margin-bottom: 40px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.works-cards-container {
+  width: 90%;
+  margin-top: 40px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.work-card {
+  max-width: 369px;
+  margin-left: 5%;
+  margin-right: 5px;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  text-align: center;
+  border: 1px solid #d0d9d4;
+  margin-bottom: 15px;
+  align-self: center;
+}
+
+.project-image {
+  height: 252px;
+  background-color: #ebf0ee;
+}
+
+.recent-works-frameworks {
+  display: flex;
+  justify-content: space-evenly;
+  margin-bottom: 40px;
+}
+
+.frameworks-item {
+  padding: 12px 12px;
+  font-family: 'Inter', sans-serif;
+  background: #ebf0ee;
+  margin-right: 5px;
+}
+
+.btn-align {
+  margin-bottom: 10px;
+}
+
+/* About me */
+.about-me {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+}
+
+.about-me p {
+  font-family: 'Inter', sans-serif;
+  font-size: 20px;
+  margin-bottom: 30px;
+}
+
+.about-me-headline {
+  width: 100%;
```

Comment:
- [ ] Well done on adding and styling about me section but please stick as much as you can following [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468).

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/158911920-4244e6b4-7da4-4559-9167-875ae3dfcec4.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/158911946-1be2c916-35e1-43eb-b6cb-9dbf598640bd.png)